receiver/RatingReceiver.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+import websockets
+import json
+
+async def listen_for_new_ratings():
+    async with websockets.connect("ws://localhost:8765") as websocket:
+        while True:
+            rating_data = json.loads(await websocket.recv())
+
+            print(f"Rating data has been received: {rating_data}")
+
+asyncio.run(listen_for_new_ratings())
```

Comment:
Leaving lines in the main scope is a bad idea. 
Code should always be wrapped in some sort of code block (`main`, `def`, etc...).
Also, always end your files with a new line.